browser-extensions/common/js/lib/challenges.js

```diff
@@ -2176,11 +2122,9 @@ function challenge_tourist(data, params) {
     return update_data_object(o)
 }
 
-// Just return true for now
-// Return true if the athlete id for this page match what is stored in the user data
+// Return true if the athlete id for this page matches what is stored in the user data
 function is_our_page(data) {
-  // return has_user_data_athlete_id(data) && has_this_athlete_id(data) && get_user_data_athlete_id(data) == get_this_athlete_id(data) 
-  return (data.info !== undefined && data.info.is_our_page)
+  return data?.info?.is_our_page ?? false
```

Comment:
Getting fancy with Javascript now :)

Reply:
Ha. It was this extension that encouraged me to learn the language (https://exercism.org/tracks/javascript), so thank you!